apps/docs/src/pages/components/header.mdx

```diff
@@ -0,0 +1,279 @@
+import { Callout } from "nextra/components";
+import { headerClasses, HvHeader } from "@hitachivantara/uikit-react-core";
+
+import Playground from "@docs/components/code/Playground";
+import { Description } from "@docs/components/page/Description";
+import { Page } from "@docs/components/page/Page";
+import { getComponentData } from "@docs/utils/component";
+
+export const getStaticProps = async ({ params }) => {
+  const meta = await getComponentData("Header", "core", headerClasses, [
+    "Actions",
+    "Brand",
+    "Navigation",
+  ]);
+  return { props: { ssg: { meta } } };
+};
+
+<Description />
+
+<Page>
+
+### Usage
+
+The example below shows a simple header with a brand, navigation, and actions. If you resize the window to a smaller size, the navigation will be hidden and a menu button will appear.
+
+```tsx live
+import { useState } from "react";
+
+export default function Demo() {
+  const width = useWidth();
+  const [selected, setSelected] = useState<string>("2");
+
+  const handleChange: HvHeaderNavigationProps["onClick"] = (
```

Comment:
can we inline this? it's only used in 1 place & no need for `HvHeaderNavigationProps["onClick"]`